RedHat-UX/red-hat-design-system
# fix: add tsconfig to pfeDevServerConfig

## What I did

1. Add tsconfig to pfeDevServerConfig in `web-dev-server.config.js`
2. Add tsconfig to pfeDevServerConfig in `web-test-runner.config.js`
3. Unpin'd pfe-tools in package.json. Allowing versions > next.29 to be installed.

Closes https://github.com/patternfly/patternfly-elements/pull/2167

## TODO:
- [x] Check to see if we can go back to unpin'd version of pfe-tools, allowing versions >29 to install. - verified in 136270f4a473efa7da3362471ca68883cfcb4cd8

## Testing Instructions

1. Checkout branch
4. Refresh dependencies `npm run clean:nuke`
5. `npm ci`
6. `npm start`
7. Check that components are working correctly along with decorators.

## Notes to Reviewers

Please make sure to look at any component using `@colorContext`, `@state`, `@query`, as those were identified as having the issues.
